data_converter/settings.py

```diff
@@ -61,13 +63,55 @@
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# CORS settings
+# https://github.com/adamchainz/django-cors-headers
+
+CORS_ORIGIN_WHITELIST = [
+    'https://dataocean.us',
+    'http://127.0.0.1:8000',
```

Comment:
move this to settings_local and add localhost for developing to test server
```suggestion
    'http://127.0.0.1:8000',
    'http://127.0.0.1:3000',
    'http://localhost:3000',
```